src/com/owncloud/android/lib/resources/files/RemoteFile.java

```diff
@@ -33,132 +33,143 @@
 import com.owncloud.android.lib.common.network.WebdavEntry;
 
 /**
- *  Contains the data of a Remote File from a WebDavEntry
- * 
- *  @author masensio
+ * Contains the data of a Remote File from a WebDavEntry
+ *
+ * @author masensio
  */
 
 public class RemoteFile implements Parcelable, Serializable {
 
-    /** Generated - should be refreshed every time the class changes!! */
+    /**
+     * Generated - should be refreshed every time the class changes!!
+     */
     private static final long serialVersionUID = 3130865437811248451L;
```

Comment:
As the comment above says: "it should be refreshed every time the class changes" but has not been updated here